packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -2392,9 +2299,7 @@ class _EmptySliderComponentShape extends SliderComponentShape {
 
 /// The default shape of a [Slider]'s thumb.
 ///
-// ignore: deprecated_member_use_from_same_package
-/// If [useV2Slider] is true, then there is a shadow for the resting and
-/// pressed state.
+/// There is a shadow for the resting and pressed state.
```

Comment:
resting, pressed, hovered, and focused state

Reply:
Done